pkg/virt-launcher/virtwrap/converter/s390x.go

```diff
@@ -39,7 +43,7 @@ func (archConverterS390X) addGraphicsDevice(vmi *v1.VirtualMachineInstance, doma
 	}
 }
 
-func (archConverterS390X) scsiController(c *ConverterContext, driver *api.ControllerDriver) api.Controller {
+func (archConverterS390X) scsiController(_ *ConverterContext, driver *api.ControllerDriver) api.Controller {
```

Comment:
What is this change needed for? As far as i can see it should just result in a linter error?

Reply:
Just to align with others places (i.e https://github.com/kubevirt/kubevirt/blob/07db79c2ce3115ecfdb1d186536d837aaa14bcf8/pkg/virt-launcher/virtwrap/converter/s390x.go#L31)

Reply:
Here the converter context is not used as well, which is why it has no variable name associated.
Here as well not giving the argument a name signals that it is not used by the actual implementation.
Though to be fair as long as the linter is happy i don't care about that rule.